From eaecb090f6f27ce7e8cef08517fbcd7f4abc554b Mon Sep 17 00:00:00 2001
From: Wang Hui <Hui.Wang@windriver.com>
Date: Wed, 27 May 2009 09:58:37 +0800
Subject: [PATCH 58/77] imx27_mach/pmic: Rename a function to avoid declaration conflict

The old function name has been declared with a different prototype
in gpio.h and these declared functions will be used by all mxc
platforms. So we must rename the function here to avoid conflict.

Signed-off-by: Wang Hui <Hui.Wang@windriver.com>
---
 arch/arm/mach-mx2/mx27ads.c |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm/mach-mx2/mx27ads.c b/arch/arm/mach-mx2/mx27ads.c
index 1f54430..291956d 100644
--- a/arch/arm/mach-mx2/mx27ads.c
+++ b/arch/arm/mach-mx2/mx27ads.c
@@ -773,7 +773,7 @@ static struct mxc_spi_master mx27ads_spi_2_data = {
 };
 #endif  /* CONFIG_SPI_MXC_SELECT3 */
 
-static int gpio_pmic_active(void)
+static int pmic_gpio_init(void)
 {
 	mxc_gpio_mode(PC15_PF_TOUT);
 
@@ -781,7 +781,7 @@ static int gpio_pmic_active(void)
 }
 
 static struct pmic_platform_data mx27ads_pmic_irq = {
-	.init = gpio_pmic_active,
+	.init = pmic_gpio_init,
 	.exit = NULL
 };
 
-- 
1.6.3.1

